packages/core/test/invoke.test.ts

```diff
@@ -2048,20 +2069,23 @@ describe('invoke', () => {
       actor.subscribe({
         complete: () => {
           done();
-        }
+        },
+        error: () => {}
```

Comment:
this looks like an unrelated change - and one that could suppress legit errors in the future